sdk/textanalytics/azure-ai-textanalytics/src/test/resources/session-records/recognizeLinkedEntitiesForBatchInput.json

```diff
@@ -1,27 +1,26 @@
 {
   "networkCallRecords" : [ {
     "Method" : "POST",
-    "Uri" : "https://javatextanalyticstestresources.cognitiveservices.azure.com/text/analytics/v3.0-preview.1/entities/linking",
+    "Uri" : "https://westus2.ppe.cognitiveservices.azure.com/text/analytics/v3.0//entities/linking",
```

Comment:
extra `/` .
Also, do we need to update all the recordings?

Reply:
extra / should be fine.

There is a lot of place involving renaming, such as, `score` to `confidenceScore`